tensorflow/tools/tf_sig_build_dockerfiles/devel.usertools/gpu.bazelrc

```diff
@@ -86,12 +86,12 @@ build:build_event_export --build_event_json_file=/tf/pkg/bep.json
 build:rbe --config=rbe_linux_cuda
 
 # For continuous builds
-test:pycpp_filters --test_tag_filters=-no_oss,-oss_excluded,-oss_serial,-benchmark-test,-v1only,gpu,-no_gpu,-no_gpu_presubmit,-no_cuda11,-cuda-only
-test:pycpp_filters --build_tag_filters=-no_oss,-oss_excluded,-oss_serial,-benchmark-test,-v1only,gpu,-no_gpu,-no_gpu_presubmit,-no_cuda11,-cuda-only
+test:pycpp_filters --test_tag_filters=gpu,-no_oss,-oss_excluded,-oss_serial,-benchmark-test,-v1only,-no_gpu,-requires-gpu:2,-multi-gpu,-no_gpu_presubmit,-tpu,-cuda-only,-oneapi-only
+test:pycpp_filters --build_tag_filters=gpu,-no_oss,-oss_excluded,-oss_serial,-benchmark-test,-v1only,-no_gpu,-requires-gpu:2,-multi-gpu,-no_gpu_presubmit,-tpu,-cuda-only,-oneapi-only
 test:pycpp_filters --test_lang_filters=cc,py --test_size_filters=small,medium
-test:pycpp --config=pycpp_filters -- //tensorflow/... -//tensorflow/compiler/tf2tensorrt/... -//tensorflow/core/tpu/... -//tensorflow/lite/... -//tensorflow/tools/toolchains/... -//tensorflow/dtensor/python/tests:multi_client_test_2gpus -//tensorflow/dtensor/python/tests:multi_client_test_nccl_2gpus -//tensorflow/dtensor/python/tests:multi_client_test_nccl_local_2gpus -//tensorflow/python/distribute/experimental:multi_worker_mirrored_strategy_test_2gpus
+test:pycpp --config=pycpp_filters -- //tensorflow/... -//tensorflow/compiler/tf2tensorrt/... -//tensorflow/core/tpu/... -//tensorflow/lite/... -//tensorflow/tools/toolchains/...
```

Comment:
Is the updated `pycpp_filters` strong enough to filter out `//tensorflow/dtensor/python/tests:multi_client_test_2gpu`?

https://github.com/ROCm/tensorflow-upstream/blob/491f087ac04c9ec826794ab9d71fe2185c43c462/tensorflow/dtensor/python/tests/BUILD#L497-L539